app/views/admin/users/index.slim

```diff
@@ -41,12 +41,14 @@ header.main-content__header role="banner"
         method: :post                                                               \
       )
 
+  - registrations_message = Settings.registrations_enabled? ? "Disable registrations" : "Enable registrations"
   div style="margin-left: 10px"
-    = link_to(                                                                             \
-        Settings.registrations_enabled? ? "Disable registrations" : "Enable registrations", \
-        admin_users_toggle_registrations_path,                                             \
-        class: "button",                                                                   \
-        method: :post                                                                      \
+    = link_to(                                                                                              \
+        registrations_message,                                                                              \
+        admin_users_toggle_registrations_path,                                                              \
+        class: "button",                                                                                    \
+        method: :post,                                                                                      \
+        data: {confirm: "WARNING! PERIGO!\nYOU ARE GOING TO #{registrations_message.upcase}\nARE YOU SURE?"} \
```

Comment:
This isn't guaranteed. If I open/close before you do, the toggle won't do what you think it will. Maybe we should change this to 2 different endpoints that are idempotent. What do you think?

Reply:
@justmendes check #123

Reply:
@justmendes check how it looks now.

Reply:
I'm still a fan of "I hereby declare I know what I'm doing" but I'll take it 😄

Reply:
Extra spicy message 🥉 ✔️

Reply:
I wonder if we can meme better?

Reply:
Hm nah, this is peak.

Reply:
Sincerely.